testgrid/specs/full.yaml

```diff
@@ -1744,3 +1744,79 @@
   unsupportedOSIDs:
   - ol-9 # containerd < 1.6 is not supported on rhel 9 variants
   - rocky-91 # containerd < 1.6 is not supported on rhel 9 variants
+- name: "containerd upgrade from 1.2.x to 1.4.x"
```

Comment:
have we not tests that we are updating more than 2 versions which will start to fail after we get this one merged?

Reply:
If we have them they will fail and then we gonna address them.